main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+    "context"
+    "flag"
+    "io/ioutil"
+
+    log "github.com/sirupsen/logrus"
+    cumuluspeer "github.com/ubclaunchpad/cumulus/cumuluspeer"
+    pstore "github.com/libp2p/go-libp2p-peerstore"
+)
+
+func main() {
+    log.Info("Starting Cumulus Peer")
+
+    // Get and parse command line arguments
+    // targetPeer is a Multiaddr representing the target peer to connect to
+    // when joining the Cumulus Network.
+    // port is the port to communicate over (defaults to peer.DefaultPort).
+    // ip is the public IP address of the this host.
+    targetPeer := flag.String("t", "", "target peer to connect to")
```

Comment:
If we know the IP and the port of the multiaddr can we infer the rest? A multiaddr as a CLI flag is an awkward ui